#
# Enumeration of available INS Modes to command the unit to
#
# ins_mode_control_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 INS_MODE_0_NO_CHANGE    = 0 #  No change to the current INS mode
	uint8 INS_MODE_1_STANDBY      = 1
	uint8 INS_MODE_2_COARSE_LEVEL = 2
	uint8 INS_MODE_3_UNAIDED_NAV  = 3
	uint8 INS_MODE_4_AIDED_NAV    = 4
	uint8 INS_MODE_9_IBIT         = 9
